pegasus/sites.v3/hourofcode.com/views/theme_common_head_after.haml

```diff
@@ -1,3 +1,5 @@
+=view :theme_google_analytics
```

Comment:
Is there a reason this is first in the file and not after theme_common_head_after.js?